docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+---
+version: '3'
+
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_PASSWORD: $POSTGRES_PASSWORD
+      POSTGRES_USER: django
+    ports:
+      - "5432:5432"   
+  nginx:
+    image: nginx
+    ports:
+      - "8080:80"
+    volumes:
+      - ./docker/nginx/zfaucet.conf:/etc/nginx/conf.d/default.conf
+      - static_volume:/usr/share/nginx/html/static
+    depends_on:
+      - zfaucet
+  zfaucet:
+    build: .
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+      - zcashd
+    volumes:
+      - .:/app
+      - static_volume:/home/zcashd/zfaucet/faucet/static
+    environment:
+      DJANGO_ENVIRONMENT: $ENVIRONMENT
+      DJANGO_POSTGRESQL_PASSWORD: $POSTGRES_PASSWORD
+      DJANGO_POSTGRESQL_USER: django
+      DJANGO_POSTGRESQL_HOST: $POSTGRES_HOST
+      RPCUSER: $RPCUSER
+      RPCPASSWORD: $RPCPASSWORD
+      ZCASH_NETWORK: http://zcashd:18232
+  zcashd:
+    build:
+      context: docker/zcashd
+    command: zcashd
+    ports:
+      - "18232:18232"
+    volumes:
+      - $ZCASHD_DATADIR:/home/zcashd/.zcash
+      - $ZCASHD_PARMDIR:/home/zcashd/.zcash-params
+
+volumes:
+  static_volume:
```

Comment:
I'm curious what it means to have a key here (`static_volume`) with no value. What does this do?

Reply:
https://docs.docker.com/compose/compose-file/#volume-configuration-reference

Long story short, this is a way to attach a volume to the containers that isn't an existing host file path. It's "private" to the containers (but obviously still accessible if you really want to). Declaring the keyword makes it available for use in the service definitions.

In the instance, it's so django can "collect" the static files to a location that the nginx container can access for serving.

We certainly can use an existing host path, but I personally like it being self contained.